### DISCUSSION TITLE
posts/java/TOTP

### DISCUSSION THREAD

**Discussion by giscus** [dreamhunter2333/dreamhunter-blog#22] [General]
# posts/java/TOTP

你指尖跃动的电光，是我此生不变的信仰

https://dreamhunter2333.com/posts/java/TOTP.html

**Comment by dreamhunter2333** [comment]
> 迁移数据
> "createdAt": "2023-06-13T18:04:48.301Z",
> "nick": "Anonymous",
> "ua": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36"

`totp = pyotp.TOTP('svcgif66')\ntotp.now()`
